log/coder_internal_test.go

```diff
@@ -221,6 +238,97 @@ func TestCoder(t *testing.T) {
 	})
 }
 
+//nolint:paralleltest // We need to replace a global timeout.
+func TestCoderRPCTimeout500MS(t *testing.T) {
```

Comment:
Nit: do we have to include the timeout in the naming?

Reply:
We don't, wanted to be explicit since we set it in the parent and use it in subtests, so that it's clear you shouldn't modify it without taking all tests into account. I'll leave a comment instead.